onig/src/names.rs

```diff
@@ -169,13 +169,13 @@ mod tests {
     #[test]
     fn test_regex_names() {
```

Comment:
Could you mark this test as `#[ignore]` rather than commenting out parts of it. I've created an issue (#100) for you to link to in a FIXME comment too.